eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -109,7 +109,7 @@ library
         transformers,
         tree-diff,
         vector,
-        vector-map
+        vector-map ^>=1.1
```

We avoid speculative upper bounds for non era related packages
```suggestion
        vector-map >=1.1
```